### DISCUSSION TITLE
low listing performance of Tailwind

### DISCUSSION THREAD

**Discussion by musen34** [tailwindlabs/tailwindcss#7831] [Help]
I want to make pure CSS output, because when I use the tailwind, my website is scrolling low on the legacy devices.

Does anyone have an idea to optimize it?

**Comment by adamwathan** [comment]
Hey! As long as you aren't using CSS modules or Svelte/Vue `<style>` blocks, try this:

```js
// tailwind.config.js
module.exports = {
  experimental: {
    optimizeUniversalDefaults: true
  },
  // ...
}
```

Here's another discussion with some more info: https://github.com/tailwindlabs/tailwindcss/discussions/7317#discussioncomment-2107898

Hoping to make this the default soon, just a bummer that the approach is not compatible with how build tools handle CSS modules and Svelte/Vue component `<style>` blocks.

**Reply by musen34** [reply]
> Hey! As long as you aren't using CSS modules or Svelte/Vue `<style>` blocks, try this:
> 
> ```js
> // tailwind.config.js
> module.exports = {
>   experimental: {
>     optimizeUniversalDefaults: true
>   },
>   // ...
> }
> ```
> 
> Here's another discussion with some more info: [#7317 (comment)](https://github.com/tailwindlabs/tailwindcss/discussions/7317#discussioncomment-2107898)
> 
> Hoping to make this the default soon, just a bummer that the approach is not compatible with how build tools handle CSS modules and Svelte/Vue component `<style>` blocks.

Thanks for your reply @adamwathan !

I used that before and its resolved some problems but not that:) I have found something that when I use opacity, it's increasing render effort of browser. Filters also the increasing browser effort but Im founding these when I make test and thats too big effort for me:) please let me your know-how about to this subject.. I want to increase speed of bulk content loads/scrolls on also legacy devices.. Because I can do with pureCSS and its working properly on legacy devices than tailwind exports:/
